stories/Components/Forms/Dropdowns/Multiselect/multiselect.scss

```diff
@@ -165,6 +187,17 @@
         right: inherit;
       }
     }
+    &.open {
+      .options {
+        .radio {
+          .radio_border {
```

Comment:
@Ashish-srij Why so much nesting? Can we simply write this as 
```
&.open {
  .options {
    .radio_border {
```

Reply:
@surbhi90 Done.